swift/apple/FirezoneNetworkExtension/Adapter.swift

```diff
@@ -136,7 +136,13 @@ class Adapter {
            let encoded = try? JSONEncoder().encode(resolvers),
            let jsonResolvers = String(data: encoded, encoding: .utf8)?.intoRustString() {
 
-          session?.setDns(jsonResolvers)
+          do {
+            session?.setDns(jsonResolvers)
+          } catch let error {
```

Comment:
```suggestion
          } catch {
```

`error` is already provided

Reply:
Where? I just copied this from another `do catch` block. Is `error` implicitly defined within a `catch`?

Reply:
Yeah, it is - that one must have been leftover from Earlier Times™